test/javascripts/acceptance/admin-users-list-test.js.es6

```diff
@@ -8,3 +8,42 @@ QUnit.test("lists users", async assert => {
   assert.ok(exists(".users-list .user"));
   assert.ok(!exists(".user:eq(0) .email small"), "escapes email");
 });
+
+QUnit.test("switching tabs", async assert => {
+  const activeUser = "<small>eviltrout@example.com</small>";
```

Comment:
Weird, you are calling `text` and it's returning the `<small>` tags? I thought it only ever returned text.

Since this is the case, think it would be better if your selector was `... .email small` instead so you can compare the email address without markup.

Reply:
This is done this way because this is escaped and I would need to htmlilfy it first before being able to search inside it. See the existing test:

```
QUnit.test("lists users", async assert => {
  await visit("/admin/users/list/active");

  assert.ok(exists(".users-list .user"));
-->  assert.ok(!exists(".user:eq(0) .email small"), "escapes email");
});
```

Reply:
Understood, thanks for clarifying!